pds4indextools/template_pds.xml

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.sch"
+    schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+$IF(Product_Ancillary)   
+<Product_Ancillary xmlns="http://pds.nasa.gov/pds4/pds/v1"
+ xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:schemaLocation="http://pds.nasa.gov/pds4/pds/v1 https://pds.nasa.gov/pds4/pds/v1/PDS4_PDS_1L00.xsd">
+$END_IF
+$IF(Product_Metadata_Supplemental)   
```

Comment:
I'm guessing that either `Product_Ancillary` is True or `Product_Metadata_Supplemental` is True, but not neither and not both. You could simplify this by simply testing for one of them and using `$ELSE`. Or maybe add tests for both and neither just to catch the case where the template is being fed bad metadata?

Reply:
I tried using $ELSE during development, and I got this version working faster than I got $ELSE working. Currently the command `--generate-label` requires you to choose either or, and sets that one to True.

Reply:
I am going to mark this as acknowledged. In the next round of changes to make I'll consider the above changes.